src/phoenix/trace/llama_index/callback.py

```diff
@@ -185,11 +197,11 @@ class OpenInferenceTraceCallbackHandler(BaseCallbackHandler):
 
     def __init__(
         self,
-        callback: Optional[Callable[[List[Span]], None]] = None,
```

Comment:
removed the callback because the buffer is now a dict instead of list

Reply:
We need to explicitly track that we need to handle and give agency over  the internal memory - we need to track this.

Reply:
actually i can put this back by making the trade-off from a different angle